### DISCUSSION TITLE
In React for the CSS properties to work, I have to save tailwind.config.js everytime

### DISCUSSION THREAD

**Discussion by zadigg** [tailwindlabs/tailwindcss#5570] [Help] [answered]
It does work perfectly in Next.js, but I don't know what the problem is with React, 
For Example: 
`<p className="bg-red-500">hello world</p>`
for the above code to work I have to save tailwind.config.js 
if I add a new property I have to save it again 

This is my package.json if it helps

`{
  "name": "my-app",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@craco/craco": "^6.3.0",
    "@testing-library/jest-dom": "^5.11.4",
    "@testing-library/react": "^11.1.0",
    "@testing-library/user-event": "^12.1.10",
    "autoprefixer": "^9",
    "postcss": "^7",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-scripts": "4.0.3",
    "tailwindcss": "npm:@tailwindcss/postcss7-compat",
    "web-vitals": "^1.0.1"
  },
  "scripts": {
    "start": "craco start",
    "build": "craco build",
    "test": "craco test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}

`
Thanks

**Comment by simonswiss** [comment, marked as answer]
Try adding `TAILWIND_MODE=watch` to your `start` script:

```diff
- "start": "craco start",
+ "start": "TAILWIND_MODE=watch craco start",
```

Hopefully that helps!

You can read more about this here:

https://tailwindcss.com/docs/just-in-time-mode#troubleshooting

Hope it helps!

**Reply by zadigg** [reply]
Thanks man, It works!!!
